TestFoundation/TestNSTextCheckingResult.swift

```diff
@@ -18,7 +18,7 @@
 #endif
 
 class TestNSTextCheckingResult: XCTestCase {
-    static var allTests: [(String, TestNSTextCheckingResult -> () throws -> Void)] {
+    static var allTests: [(String, (TestNSTextCheckingResult) -> () throws -> Void)] {
```

Comment:
Was this change also necessary? Just want to make sure this didn't accidentally slip in.